src/components/MobileNavBar/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
This file should be within the `src/components/NavBar` directory. Remember: We're trying to treat the NavBar and the MobileNavBar as the same component. Unless, of course, that changed without anybody telling me. But regardless, since they share practically the same function, they should be grouped. I would put this file in `src/components/NavBar/mobile.tsx`.

Reply:
Well they're separate react components. One's always at the top of the page, and the other's always at the bottom of the page, so I split them up into 2 components. But putting them in the same folder makes sense 👍